DDR4 OC Guide.md

```diff
@@ -183,11 +184,12 @@ Sometimes the Thaiphoon report won't tell you the IC or it may misidentify the I
   * This is the code for dual rank Micron produced in October 2018.
 * [Source](http://www.xtremesystems.org/forums/showthread.php?285750-Interesting-memory-deals-thread&p=5230258&viewfull=1#post5230258)
 
-### A Note on Ranks and Density
+### A Note on Logical Ranks and Density
 * Single rank sticks usually clock higher than dual rank sticks, but depending on the benchmark the performance gain from rank interleaving<sup>1</sup> can be significant enough to outperform faster single rank sticks. [This can be observed in both synthetics and games](https://kingfaris.co.uk/ram).
    * On recent platforms (Comet Lake and Zen3), bios support for dual rank has improved greatly. On many Z490-boards dual rank Samsung 8Gb B-die (2x16GB) will clock just as high as single-sided B-die, meaning you have all the performance gains of rank interleaving without any downsides.
    * <sup>1</sup>Rank interleaving allows the memory controller to parallelize memory requests, for example writing on one rank while the other is refreshing. The impact of this is easily observed in AIDA64 copy bandwidth. From the eyes of the memory controller, it doesn't matter whether the second rank is on the same DIMM (two ranks on one DIMM) or a different DIMM (two DIMM in one channel). It does, however, matter from an overclocking perspective when you consider memory trace layouts and BIOS support.
 * Density matters when determining how far your ICs can go. For example, 4Gb AFR and 8Gb AFR will not overclock the same despite sharing the same name. The same can be said for Micron Rev.B which exists as both 8Gb and 16Gb. The 16Gb chips overclock better and are sold as both in 16GB and 8GB capacities. The 8GB sticks have their SPD modified and can be found in higher-end Crucial kits (BLM2K8G51C19U4B).
+* As the count of ranks total in a system increases, so does the difficulty of scheduling info by the memory controller. This usually means that more memory ranks will a higher voltage, especially VCCSA on Intel and VSOC on AMD.
```

Comment:
As the number of memory banks increased, addressing and simultaneously reading from and writing to each address becomes harder. More memory banks means more bank groups, and so the signal that is sent to the address decoder is gonna need to be more specific and thus harder to run. Running at a higher voltage makes the distinction between different signals easier, which is one of the reasons higher voltage helps. I actually prefer your revision though, so that's ideal.